pystorm/component.py

```diff
@@ -470,11 +474,12 @@ def run(self):
             try:
                 self._run()
             except StormWentAwayError:
-                log.info('Exiting because parent Storm process went away.')
```

Comment:
Should probably change this one back, since if you're using `StormHandler`, this will never be able to actually get logged anyway.